template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/views/RdsView.java

```diff
@@ -27,43 +28,71 @@
 
     private static final String SSL_OPTIONS_WITHOUT_CERTIFICATE_FILE_PATH = "sslmode=verify-full&sslrootcert=";
 
-    private final String connectionURL;
+    private String connectionURL;
```

Comment:
It is not fortunate that we are losing the `final` property of all the instance fields due to the extracted `setup()` logic.